src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.Log.cs

```diff
@@ -12,17 +12,18 @@ internal sealed partial class BuildUpToDateCheck
         private sealed class Log
         {
             private readonly TextWriter _writer;
-            private readonly LogLevel _requestedLogLevel;
             private readonly Stopwatch _stopwatch;
             private readonly TimestampCache _timestampCache;
             private readonly string _fileName;
             private readonly ITelemetryService _telemetryService;
             private readonly UpToDateCheckConfiguredInput _upToDateCheckConfiguredInput;
 
+            public LogLevel Level { get; }
```

Comment:
Should we make the log level public ? Where are you using it outside this class ?

Reply:
Check this commit:

https://github.com/dotnet/project-system/commit/e94ded2f86b95c215baa154f8b93d3e5f6dae6ad